services/web/pkg/config/defaults/defaultconfig.go

```diff
@@ -47,9 +47,34 @@ func DefaultConfig() *config.Config {
 					ResponseType: "code",
 					Scope:        "openid profile email",
 				},
-				Apps: []string{"files", "search", "preview", "text-editor", "pdf-viewer", "external", "user-management"},
+				Apps: []string{"files", "search", "text-editor", "pdf-viewer", "external", "user-management"},
+				ExternalApps: []config.ExternalApp{
+					{
+						ID:   "preview",
+						Path: "web-app-preview",
+						Config: map[string]interface{}{
+							"mimeTypes": []string{
+								"image/gif",
+								"image/png",
+								"image/jpeg",
+								"text/plain",
```

Comment:
Please remove `text/plain`, otherwise the preview app will be used for rendering text files. While that's certainly possible I don't think that it makes much sense from a UX perspective. But could be discussed separately 😅

Reply:
Yeah, copy/paste error. :see_no_evil: